torrt/rpc/transmission.py

```diff
@@ -111,15 +113,29 @@ def method_get_torrents(self, hashes: List[str] = None) -> List[dict]:
 
         for torrent_info in result['torrents']:
             self.normalize_field_names(torrent_info)
+            torrent_info['excluded_files'] = self.__get_unwanted_files(torrent_info)
 
         return result['torrents']
 
-    def method_add_torrent(self, torrent: bytes, download_to: str = None) -> Any:
-
+    def method_add_torrent(self, torrent: bytes, download_to: str = None, excluded_files: List[str] = None) -> Any:
         args = {
             'metainfo': base64encode(torrent).decode(),
         }
 
+        if excluded_files:
+            # REVIEW: I don't have good idea on how not to parse torrent again.
+            # We can pass list of indices, so we don't need to parse torrent again.
+            # But I think list of file names is more generic then list of file indices
+            # and RPC implementation must adapt the list accordingly
+            torrent = parse_torrent(torrent)
+            excluded_indices = []
+            for i, f in enumerate(torrent['files']):
```

Comment:
Please stick to meaningful var names, e.g. `idx, filename`, it'd facilitate reading.

Reply:
Done.

Reply:
Yes, we'd need something more definite that dicts, maybe named tuples or even classes.
I'll think it over.